app/views/order_items/index.html.erb

```diff
@@ -1,32 +1,43 @@
-<h1>Your Cart</h1>
+<div class="panel panel-warning">
+  <div class="panel-body">
+    <h2>Your Cart  <span class="totalcart">Total:<%= @order.order_total%></span>
+</h2>
+    <!-- <% if @order.nil? %>
+      <p>No items in cart. </p>
+    <% else %> -->
+    <ul>
+      <% @order_items.each do |item| %>
+        <% product = Product.find(item.product_id) %>
+        <li>
+          <%= image_tag product.image, height: '100', width: '100', class: "productpic" %>
```

Comment:
Is this image important to the understanding of what is on the page? If so, you should include an alt attribute for screen readers/accessibility.